git_theta/checkpoints/base.py

```diff
@@ -49,6 +49,15 @@ def load(cls, checkpoint_path):
             should be numpy arrays.
         """
 
+    @classmethod
+    @abstractmethod
+    def from_framework(cls, model_dict):
+        """Convert a checkpoint from the native framework format to git-thetas."""
+
+    @abstractmethod
+    def to_framwork(self):
```

Comment:
framework

Reply:
This explains why the abstractmethod part wasn't working right lol.

Fixed